src/Core/Scheduler/Providers/3dTiles_Provider.js

```diff
@@ -133,7 +133,13 @@ $3dTiles_Provider.prototype.b3dmToMesh = function b3dmToMesh(data, layer) {
             mesh.frustumCulled = false;
             if (mesh.material) {
                 if (layer.overrideMaterials) {
-                    mesh.material = new THREE.MeshLambertMaterial(0xffffff);
+                    mesh.material.dispose();
```

Comment:
In the future, it would be cool to be able to pass parameters to the gltf parser.